### DISCUSSION TITLE
does cast kodi modify the web page when it's not being used?

### DISCUSSION THREAD

**Discussion by atomGit** [regseb/castkodi#53] [General]
from a privacy/anti-fingerprinting POV, i'm wondering if the add-on can be detected by a site when it's not being used - does it modify the page in any way?

i tried to determine this using the console and i'm not seeing any difference whether it's enabled or disabled, but i'd like confirmation

**Comment by regseb** [comment]
Cast Kodi never modifies web pages. When you don't use Cast Kodi, it doesn't do anything. And when you cast a video/music, it sends AJAX requests to retrieve the URL of the video/music in the page.

**Comment by atomGit** [comment]
thank you!
